custom-commands/drush-uli/uli

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+## Description: Log into the Drupal site with a one-time login link.
+URL=$(ddev exec drush uli | tr -d '\r')
+
+case $OSTYPE in
```

Comment:
The `launch` host command always prepends the DDEV_URL, so we cannot shortcut to the following

```
ddev launch $(ddev exec drush uli)
```